### PR TITLE
Draft consensus on "Can [TEE / MPC] be used as a private computation configuration?"

### PR DESCRIPTION
**Pull request by michaelkleber**
Draft write-up based on our discussion at the London face-to-face meeting on 2023-06-15.

### CODE REVIEW

**Review by csharrison — APPROVED** [review]
LGTM

File: design-dimensions/Dimensions-with-General-Agreement.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ This document serves to capture the dimensions with general agreement from the C
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ may include server processing dependencies, however any server architecture must achieve both a high level of security (which the editors will align with the ongoing [Threat Model](../threat-model) work) as well as ability to explain the privacy and security properties of the system to end users.
 
+More generally, the community group is willing to engage with proposals that use different mechanisms to protect data while it being processed on servers, explicitly including both Multi-Party Computation (MPC) and Trusted Execution Environment (TEE) server-side components.  These cannot be treated as "silver bullets", but rather as tools that act as individual components in larger designs, which must in turn mitigate the vulnerabilities of their components.
```
Comment (martinthomson) [inline design-dimensions/Dimensions-with-General-Agreement.md:17]:
```suggestion
More generally, the community group is willing to engage with proposals that use different mechanisms to protect data while it being processed on servers, explicitly including both Multi-Party Computation (MPC) and Trusted Execution Environment (TEE) server-side components.
These technologies each have shortcomings, but they can form part of a larger system of protections that includes mitigations for the different vulnerabilities of its constituent components.
The group could consider both technical and procedural mitigations.
```

**Review by martinthomson — APPROVED** [review]
I have something of an objection to the use of "silver bullets" or similar colloqualisms, so I suggested alternative language.  If we were indeed werewolf hunting, a lack of argent projectiles would be a serious issue; thankfully, we lead a far less exciting life.

**Comment by michaelkleber** [comment]
Thanks @martinthomson, well said.

**Comment by michaelkleber** [comment]
@AramZS or @seanturner Can you apply the "Call For Consensus" label to these PRs that document the tentative consensus from the face-to-face meeting?  Or is there another step first?

**Comment by michaelkleber** [comment]
Hey, sorry I was out on vacation during the recent PATCG meeting, but how can we push this PR forward?

**Comment by AramZS** [comment]
As discussed, these PRs have been open long enough to acquire consensus that they accurately describe our conversations.

**Pull request merged**